File: src/com/basmilius/ps/bastools/component/basSettings/BasSettingsCodeStyleScheme.java
package com.basmilius.ps.bastools.component.basSettings;

import com.basmilius.ps.bastools.component.BTCodeStyleScheme;

public class BasSettingsCodeStyleScheme extends BTCodeStyleScheme
{

	public BasSettingsCodeStyleScheme ()
	{
		super("Bas Settings", true, new BasSettingsBaseCodeStyleScheme(null));

		this.setCodeStyleSettings(this.getParentScheme(BasSettingsBaseCodeStyleScheme.class).getCodeStyleSettings());
		this.getCodeStyleSettings().setParentSettings(this.getParentScheme(BasSettingsBaseCodeStyleScheme.class).getCodeStyleSettings());
		this.getCodeStyleSettings().writeExternal(this.writeScheme());
	}

}

File: src/com/basmilius/ps/bastools/component/basSettings/BasSettingsCodeStyleScheme.kt
package com.basmilius.ps.bastools.component.basSettings

import com.basmilius.ps.bastools.component.BasToolsCodeStyleScheme

/**
 * Class BasSettingsCodeStyleScheme
 *
 * @author Bas Milius
 * @package com.basmilius.ps.bastools.component.basSettings
 */
class BasSettingsCodeStyleScheme: BasToolsCodeStyleScheme("Bas Settings", true, BasSettingsBaseCodeStyleScheme(null))
{

	/**
	 * BasSettingsCodeStyleScheme Constructor
	 *
	 * @author Bas Milius
	 */
	init
	{

		this.codeStyleSettings = this.getParentScheme(BasSettingsBaseCodeStyleScheme::class.java).codeStyleSettings
		this.codeStyleSettings.parentSettings = this.getParentScheme(BasSettingsBaseCodeStyleScheme::class.java).codeStyleSettings
		this.codeStyleSettings.writeExternal(this.writeScheme())
	}

}
